data/info

```diff
@@ -1 +1 @@
-VERSION=unknown
+VERSION=2.23.1+git355.ga9cb3ac.dirty
```

Comment:
We should fix the package build process to have a `data/info.in`, it's just annoying.